charts/tidb-operator/templates/controller-manager-deployment.yaml

```diff
@@ -50,6 +50,16 @@ spec:
         resources:
 {{ toYaml .Values.controllerManager.resources | indent 12 }}
         {{- end }}
+        readinessProbe:
+          tcpSocket:
+            port: 6060
+          initialDelaySeconds: 10
+          periodSeconds: 10
+        livenessProbe:
+          tcpSocket:
+            port: 6060
+          initialDelaySeconds: 15
+          periodSeconds: 10
```

Comment:
```suggestion
        livenessProbe:
          tcpSocket:
            port: 6060
          initialDelaySeconds: 30
          periodSeconds: 10
          failureThreshold: 10
```

Reply:
It seems no need to add readinessProbe since no service created for this deployment.